*** ./dist/vendor/assets/javascripts/bootstrap-editable.js	2013-03-10 18:11:41.000000000 +0800
--- ./vendor/assets/javascripts/bootstrap-editable.js	2013-03-10 16:39:02.000000000 +0800
***************
*** 275,285 ****
                  this.showLoading();
  
                  //standard params
!                 params = {
!                     name: this.options.name || '',
!                     value: submitValue,
!                     pk: pk 
!                 };
  
                  //additional params
                  if(typeof this.options.params === 'function') {
--- 275,296 ----
                  this.showLoading();
  
                  //standard params
!                 var data_to_send = this.options.name.split('-');
!                 if (data_to_send.length >= 2){
!                     var model = data_to_send[0];
!                     var column = data_to_send[1];
!                     params = {};
!                     var temporary = {};
!                     temporary[column] = submitValue;
!                     params[model] = temporary;
!                 }
!                 else{
!                     params = {
!                         name: this.options.name || '',
!                         value: submitValue,
!                         pk: pk
!                     };
!                 }
  
                  //additional params
                  if(typeof this.options.params === 'function') {
